timemachine/fe/mcgregor.py

```diff
@@ -358,29 +361,33 @@ def mcs(
             filter_fxn,
         )
 
-        total_nodes_visited += mcs_result.nodes_visited
+        visits_left -= mcs_result.nodes_visited
```

Comment:
Do we need both `total_nodes_visited` and `visits_left`? (e.g. could `visits_left` be replaced by `max_visits - total_nodes_visited`?)

Reply:
Wasn't needed, was just supposed to be easier to read. I replaced it with `max_visits - total_nodes_visited` and a comment in https://github.com/proteneer/timemachine/pull/1357/commits/994c5964dd7af36501f819394ffa5f40b5ed374b